requirements.txt

```diff
@@ -1,4 +1,5 @@
 # Copyright 2021 Canonical Ltd.
 # See LICENSE file for licensing details.
 
-git+https://github.com/canonical/operator/#egg=ops
+#git+https://github.com/canonical/operator/#egg=ops
+git+https://github.com/jnsgruk/operator@update-is-ready#egg=ops
```

Comment:
You seem to be in a rush. You will need another PR just to change this again in you charms :-) .